packages/apollo-server-plugin-response-cache/src/ApolloServerPluginResponseCache.ts

```diff
@@ -165,10 +172,15 @@ export default function plugin(
           async function cacheGet(
             contextualCacheKeyFields: ContextualCacheKey,
           ): Promise<GraphQLResponse | null> {
-            const key = cacheKeyString({
+            const cacheKeyData = {
               ...baseCacheKey!,
               ...contextualCacheKeyFields,
-            });
+            };
+
+            const key = options.generateCacheKey
```

Comment:
we can do this once by, higher up, having something like `const generateCacheKey = options.generateCacheKey ?? (_, key) => sha(JSON.stringify(key))`

Reply:
done - in next push